tests/test_cbv.py

```diff
@@ -6,44 +6,67 @@
 
 from fastapi_restful.cbv import cbv
 
+ONE = 1
+TWO = 2
+THREE_AS_BYTES = b'3'
+SUCCESS_CODE = 200
 
-class TestCBV:
-    @pytest.fixture(autouse=True)
+
+class AbstractTestRouter:
+    @pytest.fixture(autouse=True, scope="module")
     def router(self) -> APIRouter:
         return APIRouter()
 
-    def test_response_models(self, router: APIRouter) -> None:
-        expected_response = "home"
+    @pytest.fixture(scope="function")
+    def client(self, router: APIRouter):
+        client = TestClient(router)
+        return client
+
+    @pytest.fixture(scope="class")
+    def cbv_router(self, router: APIRouter):
+        return
 
+
+class TestResponseModels(AbstractTestRouter):
+    EXPECTED_RESPONSE = "home"
+
+    @pytest.fixture(scope="class")
+    def cbv_router(self, router: APIRouter):
         @cbv(router)
         class CBV:
             def __init__(self) -> None:
-                self.one = 1
-                self.two = 2
+                self.one = ONE
+                self.two = TWO
 
             @router.get("/", response_model=str)
             def string_response(self) -> str:
-                return expected_response
+                return TestResponseModels.EXPECTED_RESPONSE
 
             @router.get("/sum", response_model=int)
             def int_response(self) -> int:
                 return self.one + self.two
 
-        client = TestClient(router)
-        response_1 = client.get("/")
-        assert response_1.status_code == 200
-        assert response_1.json() == expected_response
+        return
 
-        response_2 = client.get("/sum")
-        assert response_2.status_code == 200
-        assert response_2.content == b"3"
+    def test_response_models(self, cbv_router: None,
+                             client: TestClient) -> None:

```

Comment:
Why has this become 2 lines all of a sudden?

Reply:
line over 90 chars